.github/workflows/update.yml

```diff
@@ -0,0 +1,40 @@
+# Updating the dataset as it was said in https://www.imf.org/external/np/res/commod/
+#Q: How often are the monthly figures updated?
+#A: Monthly data are updated on the Wednesday of the first full week of each month, so that we have time to collect all the needed data for the previous month,
+# and is posted on the website shortly afterwards.
+name: Update Dataset
+
+on:
+  schedule:
+    - cron: "0 0 * * 3"
```

Comment:
@gradedSystem any reason for this schedule? I think we want to run on monthly basis.

Reply:
In the https://www.imf.org/external/ website where the data has been pullen off there is a section of the Q&A which says [How often are the monthly figures updated?](https://www.imf.org/external/np/res/commod/#q7) so it says every: Monthly data are updated on the Wednesday of the first full week of each month

Reply:
I guess we can put the given job and change it like this: 0 0 1 * *

Reply:
ah got it. Let's do it every Wednesday then (weekly) at 23:59 so that we certainly catch the update on the same day.